i18n/en/docusaurus-plugin-content-docs/current/integration/asset-workflow.addressables.md

```diff
@@ -0,0 +1,364 @@
+  - Parameters：Retry result
+    - true: If the retry strategy is run and the retry is successful
+    - false: If the retry strategy is run and the retry is not successful finally
+
+### Encrypt and decrypt assets {#assets-addressables-crypto}
+
+:::caution
+Note that using this feature alone is weak protection for intellectual property.
+
+By using this feature, if someone obtains only encrypted assets, they will not be able to decrypt them.
+For example, even if the URL of the asset download destination is known, the asset cannot be decrypted, thus preventing misappropriation of the asset.
+
+However, if the encryption algorithm and the key used for decryption (implementation of ICryptoStreamFactory) are embedded in the application, the information necessary for decryption can be accessed and decrypted through reverse engineering.
+Therefore, for the purpose of intellectual property protection, it is necessary to prevent decryption from the application by reverse engineering in addition to this feature.
+:::
+
+Customize Addressables build scripts and ResourceProvider to encrypt and decrypt assets.
+The following classes are provided.
+
+- Build script to encrypt assets
+  - BuildScriptEncryptMode
+- ResourceProvider to decrypt assets
+  - CryptoAssetBundleProviderBase
+  - CryptoAssetBundleResource
+  - DownloadHandlerFileWithDecryption
+- Factory that creates the CryptoStream used to encrypt and decrypt the asset
+  - ICryptoStreamFactory
+
+These classes are related as follows.
+- Since the cryptographic algorithm to be used and the management of keys vary depending on the security level required by the application, an ICryptoStreamFactory is provided to provide for their processing.
+- To incorporate ICryptoStreamFactory, CryptoAssetBundleProviderBase is provided as a Base class.
+- CryptoAssetBundleProviderBase decrypts assets using ICryptoStreamFactory. (Actually CryptoAssetBundleResource and DoorloadHandlerFileWithDecryption do it)
+- BuildScriptEncryptMode encrypts using ICryptoStreamFactory if the ResourceProvider specified in the Asset Group at build execution is CryptoAssetBundleProviderBase, and Otherwise, encryption is not performed.
+
+The following is an instruction on how to apply this feature.
+
+First, set up a build script to encrypt the assets as follows.
+
+- Create a ScriptableObject from the `Extreal>Integration.AssetWorkflow.Addressables.Editor>Encrypt Build Script` in the Create Asset menu.
+- Add the ScriptableObject you created to the `Build and Play Mode Scripts` from the `AddressablesAssetSettings` inspector.
+
+This setting allows you to choose `Build>New Build>Encrypt Build Script` in the `Addressables Group` window.
+
+Next, set up the ResourceProvider that will decrypt the asset.
+ICryptoStreamFactory must be implemented first.
+Implement ICryptoStreamFactory according to the requirements of your application.
+An example implementation using AES is shown below to show the implementation image.
```

Comment:
to show the implementation image
-> to give an idea of the implementation

Reply:
Fixed!
https://github.com/extreal-dev/Extreal.Guide/pull/42/commits/46a5d9caaf19190a1628b4110e952f3ae7703baa